popup/quick-image.html

```diff
@@ -3,20 +3,24 @@
   <head>
     <meta charset="utf-8" />
     <link rel="stylesheet" href="quick-image.css" />
-    <script type="application/javascript"  src="./quick-image.js"></script>
+    <script type="application/javascript" src="./quick-image.js"></script>
   </head>
-
   <body>
     <div class="popup-content">
-      <h3 id="count">0/0</h3>
+      <h3><a id="count">0/0</a> <a id="phrase">空白</a></h3>
```

Comment:
Having both the page count and the phrase in a single line doesn't look great, lets maybe move the page count under the images.